app/code/Magento/Inventory/Model/ResourceModel/Stock/GetStockItemQuantity.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Inventory\Model\ResourceModel\Stock;
+
+use Magento\Inventory\Model\GetStockItemQuantityInterface;
+
+/**
+ * @inheritdoc
+ */
+class GetStockItemQuantity implements GetStockItemQuantityInterface
```

Comment:
I don't think we need this implementation at all. As it would be never used in the real system.

That's why I mentioned above that we no need extension point at this level